Testing/data/shapeworksTests/analyze/analysis_baseline.json

```diff
@@ -1285,5 +1285,73 @@
             "sample_3_0.vtk"
         ]
     ],
-    "groups": []
+    "groups": [],
+    "good_bad_angles": [
```

Comment:
In each domain, is the good-bad angles list in the same order as the normal particle list?

Reply:
Yes, same order as the particles in each particle file.